src/main/kotlin/com/openlattice/shuttle/Shuttle.kt

```diff
@@ -146,6 +155,14 @@ fun main(args: Array<String>) {
         cl.hasOption(CSV) -> {// get csv payload
             payload = SimplePayload(cl.getOptionValue(CSV))
         }
+        cl.hasOption(XML) -> {// get xml payload
```

Comment:
If both CSV and XML options are used, every time we will get the XML payload. I think we should check absence here also as with JDBC datasource.